app/components/Settings/cron-picker/cron-picker.jsx

```diff
@@ -80,7 +82,10 @@ export default class CronPicker extends React.Component {
                         />
                     </div>
                 </Row>
-                {ModeComponent ? <ModeComponent onChange={this.props.onChange} /> : null}
+                {ModeComponent ? (
+                    // eslint-disable-next-line
```

Comment:
why?

Reply:
if the `initialTime` is null, we want to avoid overriding the default props for the mode by passing `undefined`. Setting anything to `undefined` however causes a lint error.

Reply:
I'll defer the decision onthis one to @nicolaskruchten .

Reply:
I'm OK with this, perhaps with a comment above the disable one to explain the reason, as it's not obvious.